distributed/worker.py

```diff
@@ -1021,7 +1039,7 @@ def __init__(
         )
         self.periodic_callbacks["keep-alive"] = pc
 
-        pc = PeriodicCallback(self.find_missing, 1000)
+        pc = PeriodicCallback(self.find_missing, 1000)  # type: ignore
```

Comment:
if you run mypy with `--show-error-codes` you can use a more specific ignore:

```suggestion
        pc = PeriodicCallback(self.find_missing, 1000)  # type: ignore[arg-type]
```

Reply:
I'm unsure how I feel around this. The untyped ignore is already a lot of burden and having it fully specified would just detract from readability.